recipes/tensorflow-estimator/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set version = "1.13.0" %}
+{% set build_number = 0 %}
+
+package:
+  name: tensorflow-estimator
+  version: {{ version }}
+
+source:
+   url: https://files.pythonhosted.org/packages/bb/48/13f49fc3fa0fdf916aa1419013bb8f2ad09674c275b4046d5ee669a46873/tensorflow_estimator-{{version}}-py2.py3-none-any.whl
+   sha256: 7cfdaa3e83e3532f31713713feb98be7ea9f3065722be4267e49b6c301271419
+
+build:
+  number: {{ build_number }}
+  script:
```

need to skip Python 2.7 for Windows since [grpcio skip it](https://github.com/conda-forge/grpcio-feedstock/blob/master/recipe/meta.yaml).
```suggestion
  skip: true  # [win and py27]
  script:
```